orchestration/dagster_orchestration/hca_orchestration/solids/validate_egress.py

```diff
@@ -66,10 +67,12 @@ def base_notify_slack_of_egress_validation_results(
 ) -> str:
     gcp_env = context.solid_config["gcp_env"]
     dataset_name = context.solid_config["dataset_name"]
+    argo_id = context.solid_config["argo_id"]
 
     if validation_results.has_problems():
         message_lines = [
             f"Problems identified in post-validation for HCA {gcp_env} dataset {dataset_name}:",
+            f"Triggering Argo workflow_id: {argo_id}",
```

Comment:
🤔 It'd be useful to include this in the "has no problems" branch on line 81 below, to be able to do a similar tie out with successful validations + workflows.